docker/platypus-deps/Dockerfile

```diff
@@ -51,6 +51,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install
 RUN pip install --no-cache-dir packaging && \
     pip install --no-cache-dir pyyaml && \
     pip install --no-cache-dir setuptools && \
+    pip install --no-cache-dir xmltodict && \
```

Comment:
Perhaps this should be in a different PR, but since these new packages are being installed here, we'll probably have to add them to the ampere build script.

Reply:
Good spot - worth adding to the sapphire-rapids build too, if they're not currently available on CSD3

Reply:
Updated Ampere build script with required python dependencies for the XMLDiff test. Also switched to using an ASCII VTK output format here, to help diagnosis of issues on test failure, at the expense of test runtime. 

The ampere build has been tested successfully on CSD3.